packages/ra-ui-materialui/src/list/datagrid/expandAllButton.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+
+import { Expand } from './Datagrid.stories';
+
+describe('ExpandAllButton', () => {
+    it('should expand all rows at once', async () => {
+        const rendered = render(<Expand />);
+        const expand = () => {
+            const button = rendered.getAllByLabelText('ra.action.expand')[0];
```

Comment:
We use the `screen.getAllByLabelText` syntax in most of the codebase - that's the way react-testing-library advises to do tests

Reply:
I saw both usages in the codebase but most of the tests used `const { getAllByLabelText } = render(...)`.
I changed it in this PR.